srcd/superset/assets/src/customization/colors.js

```diff
@@ -2,23 +2,45 @@
 
 const rgba = color => `rgba(${color.r}, ${color.g}, ${color.b}, ${color.a})`;
 
-// from current superset configuration
-const mainColorSpecs = { r: 0, g: 122, b: 135, a: 1 };
+// source{d} royal (#8719cb)
+const mainColorSpecs = { r: 135, g: 25, b: 203, a: 1 };
 
-// from current superset configuration
 const palette = {
-  rausch: '#ff5a5f',
-  hackb: '#7b0051',
-  kazan: rgba(mainColorSpecs),
-  babu: '#00d1c1',
-  lima: '#8ce071',
-  beach: '#ffb400',
+  royal: "#7015a8",
```

Comment:
All these should be in single quotes.

Reply:
I know, my vs code formatter - Prettier - replaces single quotes by `"`. I'll work around it. Thank you.